src/strongbox/addon.clj

```diff
@@ -122,11 +130,20 @@
   [toc nfo]
   (merge toc nfo))
 
+(defn-spec -load-installed-addons (s/or :ok :addon/toc-list, :error nil?)
```

Comment:
this was transplanted from toc.clj - were there tests that need to be carried across?

Reply:
apparently not :(